cli/pkg/config/happy_version_lock.go

```diff
@@ -0,0 +1,92 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+type HappyVersionLockFile struct {
+	HappyVersion string
+	Path         string `json:"-"`
+}
+
+func NewHappyVersionLockFile(projectRoot string, requiredVersion string) (*HappyVersionLockFile, error) {
+
+	if projectRoot == "" {
+		return nil, errors.New("No projectRoot specified")
+	}
+
+	if requiredVersion == "" {
+		return nil, errors.New("No requiredVersion specified")
+	}
+
+	path := calcHappyVersionPath(projectRoot)
+
+	return &HappyVersionLockFile{
+		HappyVersion: requiredVersion,
+		Path:         path,
```

Comment:
Did this get fixed?

Reply:
Yeah. I excluded it in the struct definition.